reports.py

```diff
@@ -24,6 +24,49 @@
     'category: skylark > pkg build defs': 'team-Rules-Server',
 }
 
+# github handles of Bazel team members
+# TODO(aiuto): Mark users some way at github. This is not sustainable.
+BAZEL_TEAM = (
```

Comment:
do we want to expose this information?

Reply:
I guess you can get this from https://github.com/orgs/bazelbuild/people, but not everyone made their organization membership public.

Reply:
We publish name and github handles here too. https://github.com/bazelbuild/bazel/blob/master/CONTRIBUTORS

I need to reason this out. That make take a few days, because it is not urgent.
- we are not leaking any new names an github handles. Those are already visible in CONTRIBUTORS, CODEOWNERS and bazelbuild/people.
- every Google employee working on Bazel has signed a CLA and setup o
- what this does do is list github handles of people who are paid to work on bazel
- every Googler working on Bazel is already identified the second they do any work. We have all signed a license agreement as terms of employment that links our github identity to our employer. You can see it in the hovercards and user profile pages. https://github.com/aiuto

So I have to ask what makes that special?
- does that violate anyone's terms of employment?
- does that give away a trade secret- it does expose some organization structure, that many Googlers who work on Bazel are not actually part of the Bazel team.
   - does that matter?
- I think the only thing this exposes is who's performance rating will be impacted by working on Bazel.
  - is that actually a problem.
   - and does the noise on the list matter, because it misses some people and contains former team members

It would be more general to drive from a file. I will certainly do that if we generalize the tools so other teams can use it. I may now. I'll think about it.

Reply:
I turned the list into a something that can be read from a file at run time.